tests/unittest/llmapi/apps/test_chat_utils.py

```
@@ -177,3 +179,53 @@ def test_tool_message_without_tool_call_id(self, mock_mm_data_tracker):
 
         expected = {**message, "media": []}
         assert result == expected
+
+
+# ruff: noqa: E501
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)